src/models/event.ts

```diff
@@ -154,13 +154,27 @@ export interface IDecryptOptions {
     isRetry?: boolean;
 }
 
+export interface IMessageHiding {
+    reason?: string;
+}
+
 export class MatrixEvent extends EventEmitter {
     private pushActions: IActionsObject = null;
     private _replacingEvent: MatrixEvent = null;
     private _localRedactionEvent: MatrixEvent = null;
     private _isCancelled = false;
     private clearEvent?: IClearEvent;
 
+    /* Message hiding, as specified by MSC3515.
```

Comment:
wrong MSC number? I suggest links to avoid people having to convert between these manually.